src/cmd/cmd.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 var (
-	supportedSpotifyVersion = "1.1.84"
+	supportedSpotifyVersion = "1.1.90"
```

Comment:
Are you sure that these changes completely fix the error that occurs in [apply.go#L203](https://github.com/spicetify/spicetify-cli/blob/bb9d95ac1bbbb3d81ee8ec461e06f3240a439c62/src/apply/apply.go#L203)?

Reply:
The error attached did not occur for me on this patch, nor did it occur for many versions now. So I assume that it's safe to bump.